cmd/ratify/cmd/serve.go

```diff
@@ -62,6 +64,9 @@ func NewCmdServe(argv ...string) *cobra.Command {
 	flags.BoolVar(&opts.enableCrdManager, "enable-crd-manager", false, "Start crd manager if enabled")
 	flags.IntVar(&opts.cacheSize, "cache-size", httpserver.DefaultCacheMaxSize, "Cache size for the verifier http server")
 	flags.DurationVar(&opts.cacheTTL, "cache-ttl", httpserver.DefaultCacheTTL, "Cache TTL for the verifier http server")
+	// default values of "" and 0 are used to indicate that metrics are not enabled
+	flags.StringVar(&opts.metricsType, "metrics-type", "", "Metrics exporter type to use")
```

Comment:
Little bit confused on which direction we want to take. Right now if we provide a metrics type and port on the cli `serve` command, the metrics server starts up. The default value is set to be empty so that by default metrics are not emitted on the `serve` command. The value shipped with the chart is filled out for metrics type and port meaning metrics will be on but if the helm chart value is removed, then the same behavior will be exhibited as the CLI command since all we do right now is set the value of the metric type and metric port CLI flag to the helm chart equivalent values. Do you think we should have the metrics always on so that anytime someone uses the `serve` command it starts metric. I thought we would want that to be opt in

Reply:
The other option is to introduce another flag for the CLI called `metrics-enabled` which is functionally the same as the helm chart equivalent?

Reply:
I decided to add a new metrics-enabled flag